CarRental.Grpc.Producer/Program.cs

```diff
@@ -0,0 +1,48 @@
+using CarRental.Grpc.Protos;
+using CarRental.Grpc.Server.Grpc;
+using CarRental.Grpc.Server.Utils;
+using Grpc.Net.Client;
+
+
+var builder = Host.CreateApplicationBuilder(args);
+
+builder.Logging.AddConsole();
+
+builder.Services.AddSingleton<Generator>();
+
+builder.Services.AddHostedService<Producer>();
+
+builder.Services.AddSingleton(serviceProvider =>
+{
+    var grpcServiceUrl = "https://localhost:7133";
```

Comment:
Есть уверенность что адрес не изменится? Харкодить я бы не рекомендовал.
Сейчас он берется из профиля запуска.
Я бы добавил получение из конфигурации. Также можно попытаться его получить от Aspire, в данном случае это будет выглядеть так:
```suggestion
    var grpcServiceUrl = builder.Configuration.GetValue<string>("services:api:https:0")
        ?? "https://localhost:7133";
```
Получаем первый https адрес для сервиса с именем api.